src/examples/src/widgets/select/DisabledSelect.tsx

```diff
@@ -0,0 +1,14 @@
+import { create, tsx } from '@dojo/framework/core/vdom';
+import Select from '@dojo/widgets/select';
+import icache from '@dojo/framework/core/middleware/icache';
+
+const factory = create({ icache });
+const options = [{ value: 'cat' }, { value: 'dog' }, { value: 'fish' }];
+
+export default factory(function DisabledSelect() {
+	return (
+		<virtual>
+			<Select label="Disabled Select" options={options} disabled onValue={() => {}} />
```

Comment:
how come you have to pass an empty function that does nothing?

Reply:
`onValue` is required, this select is disabled, so there's no need to use it